compiler/src/main/kotlin/io/spine/protodata/renderer/InsertionPoint.kt

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright 2021, TeamDev. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Redistribution and use in source and/or binary forms, with or without
+ * modification, must retain the above copyright notice and the following
+ * disclaimer.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package io.spine.protodata.renderer
+
+/**
+ * A point is a source file, where more code may be inserted.
+ */
+public interface InsertionPoint {
+
+    /**
+     * The name of this insertion point.
+     */
+    public val label: String
+
+    /**
+     * Locates the line number where the insertion point should be added.
+     *
+     * An insertion point should only appear once in a file.
+     *
+     * @param
+     *     lines a list of code lines in a source file
+     * @return the line number at which the insertion point should be added
```

Comment:
@alexander-yevsyukov @dmdashenkov we recently had a short discussion on whether we should re-think documenting the `@return`s for Kotlin types. Alexander made a point about following the Kotlin conventions (namely, what they use in Collections) and considering each `@return` value description a sentence and completing it with a period. Please see [this conversation](https://github.com/SpineEventEngine/time/pull/112#discussion_r611079655).

I propose to settle on this approach right away in ProtoData.

@dmdashenkov @alexander-yevsyukov please feel free to comment on this.